plugin/elastic/esinfo.go

```diff
@@ -154,6 +159,56 @@ func parseESPort(infos []string) []int {
 	return getPortByPid(pid)
 }
 
+func getPortByPid(pid string) []int {
```

Comment:
this seems not related to test, if it is a feature or a fix, you should raise another PR